static/xref/.prettierrc

```diff
@@ -0,0 +1,4 @@
+{
+  "singleQuote": true,
```

Comment:
🤢... just sayin

Reply:
I know I know 😞 
There were mostly single quotes earlier in this file... ran prettier => too many changes.. hence why